somaticseq.wdl

```diff
@@ -0,0 +1,289 @@
+version 1.0
+
+import "common.wdl" as common
+
+task SomaticSeqParallelPaired {
+    input {
+        String? preCommand
+        String? installDir
+
+        File? classifierSNV
+        File? classifierIndel
+        String outputDir
+        Reference reference
+        File? inclusionRegion
+        File? exclusionRegion
+        IndexedBamFile tumorBam
+        IndexedBamFile normalBam
+        File? mutect2VCF
+        File? varscanSNV
+        File? varscanIndel
+        File? jsmVCF
+        File? somaticsniperVCF
+        File? vardictVCF
+        File? museVCF
+        File? lofreqSNV
+        File? lofreqIndel
+        File? scalpelVCF
+        File? strelkaSNV
+        File? strelkaIndel
+
+        Int threads = 1
+    }
+
+    String toolCommand = if defined(installDir)
```

Comment:
This is making things hard on ourselves to support infrastructures that don't run containers. But do we want to support those non-reproducible infrastructures anyway?